jsk_topic_tools/CMakeLists.txt

```diff
@@ -31,6 +31,21 @@ configure_file(
   ${CMAKE_CURRENT_SOURCE_DIR}/include/jsk_topic_tools/nodelet_version.h.in
   ${CATKIN_DEVEL_PREFIX}/${CATKIN_GLOBAL_INCLUDE_DESTINATION}/jsk_topic_tools/nodelet_version.h)
 
+catkin_package_xml()
```

Comment:
sorry, I think it safe to put `configure_file` after `catkin_package` so that we do not need to use `catkin_package_xml`, because I suppose `catkin_package_xml` is not designed to used in User's program. how do you think? @iory

Reply:
I see. That's good for users.
https://github.com/ros/catkin/blob/2612c4ce82a285cac13263576f8a85b8a6d6cd3d/cmake/catkin_package.cmake#L97-L100
`catkin_package` internally calls `catkin_package_xml`.

I created a PR https://github.com/jsk-ros-pkg/jsk_common/pull/1755